client/services/FeedbackGeneratorService.js

```diff
@@ -58,7 +58,8 @@ tie.factory('FeedbackGeneratorService', [
         var errorMessage = codeEvalResult.getErrorMessage();
         // We want to catch and handle a timeout error uniquely, rather than
         // integrate it into the existing feedback pipeline.
-        if (errorMessage.toString().startsWith('TimeLimitError')) {
+        if (errorMessage !== null && 
```

Comment:
Sure, that's fine. Yeah, sorry, I missed it too -- I didn't realize errorMessage was potentially nullable.